init-tools.sh

```diff
@@ -6,74 +6,18 @@ __PACKAGES_DIR=$__scriptpath/packages
 __TOOLRUNTIME_DIR=$__scriptpath/Tools
 __DOTNET_PATH=$__TOOLRUNTIME_DIR/dotnetcli
 __DOTNET_CMD=$__DOTNET_PATH/dotnet
-__DOTNET_SDK_PATH=$__TOOLRUNTIME_DIR/dotnetclisdk
-__DOTNET_SDK_CMD=$__DOTNET_SDK_PATH/dotnet
 if [ -z "$__BUILDTOOLS_SOURCE" ]; then __BUILDTOOLS_SOURCE=https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json; fi
+export __BUILDTOOLS_USE_CSPROJ=true
```

Comment:
It controls some initialization stuff in buildtools. This actually should be auto-detected but that is broken ATM